dbms/src/WindowFunctions/IWindowFunction.cpp

```diff
@@ -180,25 +186,23 @@ struct WindowFunctionLastValue final : public IWindowFunction
         : IWindowFunction(argument_types_)
     {
         RUNTIME_CHECK(argument_types_.size() == 1);
-        return_type = argument_types_[0];
+        if (argument_types_[0]->isNullable())
+            return_type = argument_types_[0];
+        else
+            return_type = std::make_shared<DataTypeNullable>(argument_types_[0]);
```

Comment:
I think you can use `return_type = makeNullable(argument_types_[0])`

Reply:
> I think you can use `return_type = makeNullable(argument_types_[0])`

okk